### PR TITLE
config: use EventsLogger=file without systemd

### PR DESCRIPTION
**Pull request by giuseppe**
if systemd is not available, use the file events logger backend.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
LGTM, and (as long as I'm careful to `rm -rf /{,var/}tmp/run-1000`) the test binary is working fine. Thanks @giuseppe for looking at this so quickly.

File: libpod/config/config.go [hunk 1]
```diff
@@ -545,5 +545,6 @@ func (c *Config) checkCgroupsAndAdjustConfig() {
 		logrus.Warningf("Alternatively, you can enable lingering with: `loginctl enable-linger %d` (possibly as root)", rootless.GetRootlessUID())
 		logrus.Warningf("Falling back to --cgroup-manager=cgroupfs")
```
Comment (rhatdan) [inline libpod/config/config.go]: This warning should be updated to say faling back to --events-logger=file

File: libpod/config/config.go [hunk 2]
```diff
@@ -545,5 +545,6 @@ func (c *Config) checkCgroupsAndAdjustConfig() {
...
 		c.CgroupManager = define.CgroupfsCgroupsManager
+		c.EventsLogger = "file"
```
Comment (mheon) [inline libpod/config/config.go]: Are we overriding the config file here? I'm a little concerned that we potentially override explicit configuration from the user.

File: libpod/config/config.go (hunk 1 above)
Comment (giuseppe) [reply libpod/config/config.go]: done

File: libpod/config/config.go (hunk 2 above)
Comment (giuseppe) [reply libpod/config/config.go]: yes, it overrides the config file here.  It works similarly to what we already do with the cgroups manager.

@vrothberg should this check be part of the default file only (`defaultConfigFromMemory`)?

Comment (vrothberg) [reply libpod/config/config.go]: I think https://github.com/containers/libpod/blob/master/libpod/config/config.go#L530 (maybe changing the functions name) would be a good place to add that if we wish to _force_ it. If we want users to be able to set it and not overwrite, `defaultConfigFromMemory`.

Comment (giuseppe) [reply libpod/config/config.go]: pushed a new version where it is done only for the default configuration

**Comment by vrothberg** [comment]
Failures looked flaky, so I restarted them.

**Comment by TomSweeneyRedHat** [comment]
LGTM, prune remote tests seem to be flaking all over....

**Pull request merged**